instance/management/commands/deprovision_buckets.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015-2019 OpenCraft <contact@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+This management command will deprovision the S3 bucket for archived instances
+"""
+
+from datetime import timedelta
+import logging
+
+from botocore.exceptions import ClientError
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from instance.models.instance import InstanceReference
+from instance.models.openedx_instance import OpenEdXInstance
+
+LOG = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    This management command will deprovision the S3 buckets of instance which
+    have been archived longer than 3 months ago.
+
+    It accepts a list of instance ref ids in case buckets ofnon-sandbox
+    instances should get archived.
+    """
+    help = (
+        'Deprovision S3 buckets for archived instances'
+    )
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--instance_ref_ids',
```

Comment:
To be a bit more POSIX compliant, how about using dashes instead of underscores in the flag name?

```suggestion
            '--instance-ref-ids',
            dest='instance_ref_ids',
```